bindings/python/README.md

```diff
@@ -0,0 +1,133 @@
+# IOTA Client Python Library
+
+## Requirements
+- Rust 1.45.0+
+- Python 3.6+
+
+## Try Run w/ Local Hornet
+
+1. Run your local Hornet
+- `$ git clone git@github.com:gohornet/hornet.git`
+- checkout `chrysalis-pt2` branch
+- Modify your `create_snapshot_alphanet.sh`, modify Line 14 to `go run ../main.go tool snapgen alphanet1 96f9de0989e77d0e150e850a5a600e83045fa57419eaf3b20225b763d4e23813 snapshots/alphanet1/full_export.bin`
+- `$ ./run_coo_bootstrap.sh `
+
+2. Build the iota-client-python library by yourself (Optional)
+- **IMPORTANT**: Enable the `bindings/python/native` in your `Cargo.toml`
+```toml
+[workspace]
+members = [
+    "iota-core",
+    "bindings/python/native",
+    "iota-client",
+    "examples",
+]
+``` 
+- `$ cargo build --release`
+- The built library is located in `target/release/`
+- On MacOS, rename `libiota_client.dylib` to `iota_client.so`, on Windows `libiota_client.dll` to `iota_client.pyd`, and on Linux `libiota_client.so` to `iota_client.so`.
```

Comment:
On Windows the file is only iota_client.dll called, without `lib` before

Reply:
Thank you for pointing this out! Modified.